<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QWidget" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>547</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#LoginPage
{
background:#FFFFFF;
border-radius:8px;
border-bottom-left-radius:8px;
border-bottom-right-radius:8px;
border:1px solid #E9EEF5;
	border-image: url(:/icons/images/background.jpg);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="spacing">
    <number>24</number>
   </property>
   <property name="leftMargin">
    <number>84</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>84</number>
   </property>
   <property name="bottomMargin">
    <number>48</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_title" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>35</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font-size:32px;
font-weight:normal;
color:rgba(0,0,0,0.85);</string>
          </property>
          <property name="text">
           <string>杏林医疗信息系统</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>9</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_error_hint" native="true">
          <property name="minimumSize">
           <size>
            <width>368</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>368</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:#FFF1F0;
border: 1px solid #FFCCC7;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="leftMargin">
            <number>16</number>
           </property>
           <property name="rightMargin">
            <number>16</number>
           </property>
           <item>
            <widget class="QLabel" name="label_account_error_left">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:1px dashed rgba(0, 0, 0, 0.45);
border-image: url(:/icons/images/icon_error.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../../res/images/login_account_error_left.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_info">
             <property name="styleSheet">
              <string notr="true">font-weight: normal;
color: rgba(0,0,0,0.65);
font-size:14px;
border:none;</string>
             </property>
             <property name="text">
              <string>账号或密码错误</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_account_error_right">
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/icons/images/login_account_error_right.png);
border:1px dashed rgba(0, 0, 0, 0.45);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>15</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>16</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 16px;
font-weight: normal;
color: #565757;</string>
            </property>
            <property name="text">
             <string>账号</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_user" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>396</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: #F7F8FA;</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="rightMargin">
              <number>16</number>
             </property>
             <property name="bottomMargin">
              <number>8</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image: url(:/icons/images/login_user.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineedit_user">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>24</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font-size: 16px;
font-weight: normal;
color: rgba(0,0,0,0.85);
background:transparent;
border:none;</string>
               </property>
               <property name="text">
                <string>张飞</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 16px;
font-weight: normal;
color: #565757;</string>
            </property>
            <property name="text">
             <string>密码</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_passwd" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>396</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#widget_passwd
{
background: #F7F8FA;
}
</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="rightMargin">
              <number>16</number>
             </property>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image: url(:/icons/images/login_passwd.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineedit_passwd">
               <property name="styleSheet">
                <string notr="true">font-size: 16px;
font-weight: normal;
color: #252525;
background:transparent;
border:none;
lineedit-password-character: 42;


</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Password</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>368</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:none;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_login">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background: #1890FF;
border-radius:2px;
font-size: 16px;
font-weight: normal;
color: #FFFFFF;
}

QPushButton:pressed{
background:#07689f;
}
</string>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>43</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_quit">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed{
background:#dde7f2;
}
QPushButton{
background: #e3f9fd;
border-radius:2px;
font-size: 16px;
font-weight: normal;
color: c4dcdf;
}
</string>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>368</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 14px;
font-weight: normal;
color: #565757;
background:none;
padding-left:64px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_create_account">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>368</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 14px;
font-weight: normal;
color: #1890FF;
background:0xdfbe6f;
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
